vector/src/main/java/im/vector/app/features/home/HomeActivity.kt

```diff
@@ -634,10 +635,18 @@ class HomeActivity :
                 launchInviteFriends()
                 true
             }
+            R.id.menu_home_qr -> {
+                launchQrCode()
+                true
+            }
             else -> false
         }
     }
 
+    private fun launchQrCode() {
+        startActivity(UserCodeActivity.newIntent(this, sharedActionViewModel.session.myUserId))
```

Comment:
OK, but this is not normal that `sharedActionViewModel.session` is public. Some shortcuts have been taken in the past :/

This is not new though, so not blocking this PR.